examples/sycl/00_pvc_gemm/00_pvc_gemm.cpp

```diff
@@ -231,11 +231,16 @@ struct ExampleRunner {
     stride_C = cutlass::make_cute_packed_stride(StrideC{}, cute::make_shape(M, N, L));
     stride_D = cutlass::make_cute_packed_stride(StrideD{}, cute::make_shape(M, N, L));
 
-    block_A.reset(static_cast<std::size_t>(M) * K * L);
-    block_B.reset(static_cast<std::size_t>(K) * N * L);
-    block_C.reset(static_cast<std::size_t>(M) * N * L);
-    block_D.reset(static_cast<std::size_t>(M) * N * L);
-    block_ref_D.reset(static_cast<std::size_t>(M) * N * L);
+    try{
+      block_A.reset(static_cast<std::size_t>(M) * K * L);
+      block_B.reset(static_cast<std::size_t>(K) * N * L);
+      block_C.reset(static_cast<std::size_t>(M) * N * L);
+      block_D.reset(static_cast<std::size_t>(M) * N * L);
+      block_ref_D.reset(static_cast<std::size_t>(M) * N * L);
+    } catch(...){
```

Comment:
It may be simpler to just update the allocation call of DeviceAllocation, rather than all of it's uses https://github.com/codeplaysoftware/cutlass-sycl/blob/69d7197e473782e86ef3825dc053ec9fc9aa3a66/tools/util/include/cutlass/util/device_memory.h#L65

Reply:
After some internal discussion and testing I have instead reverted the changes to the examples. It turns out they already print about oom if they encounter that.